src/index.html

```diff
@@ -12,7 +15,132 @@
       href="styles/index.scss"
     />
```

Comment:
The href attribute in the link tag should point to a CSS file, not an SCCSS file. The browser doesn't understand SCSS syntax. You should compile your SCSS to CSS and then link that CSS file in your HTML.